README.rst

```diff
@@ -187,11 +143,58 @@ Some things to keep in mind when building eggs for your Scrapy project:
    different user which may not have write access to certain directories. If you can, avoid writing
    to disk and always use tempfile_ for temporary files.
 
-.. _pkgutil.get_data: http://docs.python.org/library/pkgutil.html#pkgutil.get_data
-.. _tempfile: http://docs.python.org/library/tempfile.html
+.. _pkgutil.get_data: https://docs.python.org/library/pkgutil.html#pkgutil.get_data
+.. _tempfile: https://docs.python.org/library/tempfile.html
+
+
+scrapyd-client
+--------------
+
+For a reference on each subcommand invoke ``scrapyd-client <subcommand> --help``.
 
+Where filtering with wildcards is possible, it is facilitated with fnmatch_.
+The ``--project`` option can be omitted if one is found in a ``scrapy.cfg``.
+
+.. _fnmatch: https://docs.python.org/library/fnmatch.html
+
+deploy
+~~~~~~
+
+At the moment this is a wrapper around `scrapyd-deploy`_. Note that the command line options
+of this one are likely to change.
```

Comment:
This sentence was here for a while: "Note that the command line options of this one are likely to change.", but options probably havent changed for several years, so maybe makes sense to remove it? If options will change we will add documentation.

Reply:
Done in 7bf9464